### ISSUE TITLE
Some messages error with "Server response was embeds=[0]"

### ISSUE THREAD

**Issue by BrainStone** [jellysquid3/gitlab-discord-bot#10]
Some messages (from my testing it seems like it's mostly related to comments) cause this error:

```
error: Received server response 400 while executing webhook
error: Server response was embeds=[0]
error: Failed to deliver Discord message after 155ms error=Failed to POST webhook, details=The webhook returned a non-200 OK response
```

*(Specific message was cause by updating a comment)*

When multiple errors occur the first line is printed again.

**Comment by jellysquid3** [comment]
Is this reproducible with a specific event from GitLab or does it happen sporadically? I've seen this error randomly occur on my own installation but haven't been able to narrow down the specific cause.

**Comment by BrainStone** [comment]
It mostly happens when updating a comment. But it is pretty sporadic.

**Comment by jellysquid3** [comment]
After some capturing of bad events, it seems I'm failing to escape message bodies, resulting in invalid JSON being sent to Discord's servers.

**Comment by BrainStone** [comment]
Sounds like a simple fix

**Comment by rszemplinski** [comment]
I too am getting this issue and trying to fix it atm :P

**Comment by rszemplinski** [comment]
Fixed the issue and made a PR #11

**Comment by jellysquid3** [comment]
Resolved by PR #11.
